This patch fixes errors like:

  Error attempting call to get driver name: connection is shut down
  (runner-c2db3a9a-ict-1487649355-43f4611f) DBG | ... Post : unsupported protocol scheme ""
  unexpected EOF

More details:
`d.Config` is initially empty, `d.buildConfig()` must be called to load
the actual configuration. But it's called only in `Create()` method, so
in all other cases empty config is passed into goca and eventually into
goca's `Call()` method. There it goes into `xmlrpcClient` that returns
error `unsupported protocol scheme ""`. This error is not returned, but
instead logged using `log.Fatal`. And because Go is totally retarted,
this *logging* method calls `os.Exit(1)`... So the plugin process
prematurely dies and docker-machine, obviously, isn't very happy...

--- a/opennebula.go
+++ b/opennebula.go
@@ -103,6 +103,7 @@
 }
 
 func (d *Driver) setClient() error {
+	d.buildConfig()
 	return goca.SetClient(d.Config)
 }
 
@@ -365,7 +366,6 @@
 	}
 
 	// build config and set the xmlrpc client
-	d.buildConfig()
 	d.setClient()
 
 	// Create template
